### ISSUE TITLE
unserialize(): Error at offset 0 of 288 bytes

### ISSUE THREAD

**Issue by ulefebvr** [NoelDeMartin/laravel-dusk-mocking#32]
Hello, I'm  facing an issue running your package. 

Result of php artisan dusk: 

```
.E                                                                  2 / 2 (100%)

Time: 00:27.686, Memory: 49.00 MB

There was 1 error:

1) Tests\Browser\AutomationTest::test_tag_added_trigger_send_mail_action
ErrorException: unserialize(): Error at offset 0 of 288 bytes

/var/www/backend/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php:270
/var/www/backend/vendor/noeldemartin/laravel-dusk-mocking/src/Driver.php:162
/var/www/backend/vendor/laravel/framework/src/Illuminate/Support/Manager.php:191
/var/www/backend/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:338
/var/www/backend/vendor/noeldemartin/laravel-dusk-mocking/src/MockingProxy.php:74
/var/www/backend/vendor/noeldemartin/laravel-dusk-mocking/src/MockingProxy.php:48
/var/www/backend/tests/Browser/AutomationTest.php:113
/var/www/backend/vendor/laravel/dusk/src/Concerns/ProvidesBrowser.php:70
/var/www/backend/tests/Browser/AutomationTest.php:115
```

Here is the test code

```php
public function test_tag_added_trigger_send_mail_action()
    {
        $this->browse(function (Browser $browser){

            $user = User::find(1);
            $tag1 = Tag::factory()->create();
            $contact = Contact::factory()->for($user)->create();
            $model = MailTemplate::factory()->for($user)->create();

            $mail = $browser->fake(Mail::class);

            $browser->visit(new AutomationPage)
                ->createAutomation('TagAddedTrigger AddTagAction')
                ->active(true)
                ->addTrigger('tag added', ['tag' => $tag1->name, 'runs' => 'Once'])
                ->addAction('send an email', [
                    'email' => 'test@mail.com',
                    'layout' => 'Layout template 1',
                    'model' => $model->title
                ]);

            $browser->visit(new CrmPage)
                ->addTagToContact([$tag1->name], $contact->email);

            $this->artisan('schedule:run');

            $mail->assertSent(SendMail::class);
        });
    }
```

And I placed a debug in /var/www/backend/vendor/noeldemartin/laravel-dusk-mocking/src/MockingProxy.php:75

```
[2023-07-31 05:19:01] local.DEBUG: array (
  0 => '/var/www/backend/vendor/noeldemartin/laravel-dusk-mocking/src/MockingProxy.php:71@getFacadeMock',
  1 => '<html>
<head><title>405 Not Allowed</title></head>
<body>
<center><h1>405 Not Allowed</h1></center>
<hr><center>nginx</center>
</body>
</html>
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
<!-- a padding to disable MSIE and Chrome friendly error page -->
',
)
```

**Comment by NoelDeMartin** [comment]
Hey, thanks for opening an issue.

Just a heads up that I won't be able to look into this until September at least, because I'm about to go on holidays. But can you give me some details about which version of Laravel are you using and so on? Ideally, an online repository reproducing the bug would be the most useful :). I have this repository with CI configured in case you want a starting point:  https://github.com/noelDeMartin/laravel-dusk-mocking-sandbox

**Comment by NoelDeMartin** [comment]
Hey @ulefebvr, looking into this I realize I cannot reproduce it because I'm missing half of the classes (`AutomationPage`, `CrmPage`, etc.).

Can you please provide a simpler example or a failing test on CI (like [the sandbox](https://github.com/noelDeMartin/laravel-dusk-mocking-sandbox))? Otherwise, I don't think I can help you. Just by looking at the error trace I'm not sure what's the problem.

**Comment by ulefebvr** [comment]
Hello I found out the issue was coming from the fact that it inject routes. I was testing a vuejs app using Laravel Dusk, so no way, it could inject the necessary route to transfer the data back and forth.

Then I tried to import the route directly to the backend. It solve the "405 Not Allowed" issue, but I was stuck in a timeout issue. 

At the end, I chose to code a MailFaker with the data saved in a file base database.
